docs/quickstarts/sso-quickstart.md

```diff
@@ -0,0 +1,128 @@
+
+    > [!NOTE]
+    > Office Add-ins should use HTTPS, not HTTP, even when you are developing. If you are prompted to install a certificate after you run the following command, accept the prompt to install the certificate that the Yeoman generator provides.
+
+    ```command&nbsp;line
+    npm start
+    ```
+
+2. In the Office client application that opens when you run the previous command (i.e., Excel, Word or PowerPoint), make sure that you're signed in with a user that's a member of the same Office 365 organization as the Office 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso). Doing so establishes the appropriate conditions for SSO to succeed. 
+
+3. In the Office client application, choose the **Home** tab, and then choose the **Show Taskpane** button in the ribbon to open the add-in task pane. The following image shows this button in Excel.
+
+    ![Excel add-in button](../images/excel-quickstart-addin-3b.png)
+
+4. At the bottom of the task pane, choose the **Get My User Profile Information** button to initiate the SSO process. 
+
+    > [!NOTE] 
+    > If you're not already signed in to Office at this point, you'll be prompted to sign in. As described previously, you should sign in with a user that's a member of the same Office 365 organization as the Office 365 administrator account that you used to connect to Azure while configuring SSO in step 3 of the [previous section](#configure-sso), if you want SSO to succeed.
+
+5. If a dialog window appears to request permissions on behalf of the add-in, this means that SSO is not supported for your scenario and the add-in has instead fallen back to an alternate method of user authentication. This may occur when the tenant administrator hasn't granted consent for the add-in to access Microsoft Graph, or when the user isn't signed into Office with a valid Microsoft Account or Office 365 ("Work or School") account. Choose the **Accept** button in the dialog window to continue.
+
+    ![Permissions request dialog](../images/sso-permissions-request.png)
+
+6. The add-in retrieves profile information for the signed-in user and writes it to the document. The following image shows an example of profile information written to an Excel worksheet.
+
+    ![User profile information in Excel worksheet](../images/sso-user-profile-info-excel.png)
+
+## Next steps
+
+Congratulations, you've successfully created a task pane add-in that uses SSO when possible, and uses an alternate method of user authentication when SSO is not supported. To learn more about SSO configuration steps that the Yeoman generator completed automatically, and the code that facilitates the SSO process, see the [Create a Node.js Office Add-in that uses single sign-on](../develop/create-sso-office-add-ins-nodejs.md) tutorial.
```

Comment:
Should we add a section about how to update the Graph scopes of the Yo Office template? Or is the right next step for them to review the tutorial and create their own backend / scopes using that?

Reply:
We might want to call out that they can update the env file to use a different port and scope, but I worry that this might over-complicate the quick-start. Users can also update some other params as well. Here's the default .env file content before configuration:

CLIENT_ID={CLIENT_ID}
GRAPH_URL_SEGMENT=/me
NODE_ENV=development
PORT={PORT}
QUERY_PARAM_SEGMENT=
SCOPE=User.Read

Reply:
@nokafor good question. IMO, adding detailed info about the procedure for updating scopes might be a bit too much for the quick start. Maybe we could plan to create a separate article fairly soon as a follow-on to the quick start? i.e., an article that explains how to modify the project that Yo Office creates (and how to update the corresponding settings in Azure) to facilitate a different type of SSO scenario -- for example, a scenario that requires other scopes, etc. If you think this is a good idea, I can add it to the docs backlog.

For now, per @TCourtneyOwen's suggestion, I've just added a note about the **ENV** file (in the [Explore the project](https://review.docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/sso-quickstart?branch=kb-sso-quickstart#explore-the-project) section of the quick start) to call out that this type of info can be updated there. Thoughts/comments on this approach?

![image](https://user-images.githubusercontent.com/3019529/72104019-4a11d580-32df-11ea-838f-0072df9df010.png)